content/en/blog/_posts/2022-08-25-pod-security/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: blog
+title: "Pod Security Admission goes Stable"
+date: 2022-08-25
+slug: pod-security-admission-stable
+canonicalUrl: https://kubernetes.dev/blog/2022/08/25/pod-security-admission-stable/
+---
+
+ **Authors:** Tim Allclair and Sam Stoelinga (Google)
+
+The release of Kubernetes v1.25 marks a major milestone for Kubernetes out-of-the-box pod security
+controls: Pod Security admission (PSA) graduated to stable, and Pod Security Policy (PSP) has been
+removed.
+[PSP has been deprecated since Kubernetes v1.21](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/),
+but as of v1.25 it will no longer be functional.
+
+Pod Security Admission was designed as PSP's replacement, and makes it easier to enforce predefined
+[Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) by
+simply adding a label to a namespace. The Pod Security Standards are maintained by the K8s
+community, which means you automatically get updated security policies when new K8s features are
```

Comment:
```suggestion
community, which means you automatically get updated security policies when new features are
```
I'm a little confused here - whenever new features are introduced? Or when new features are introduced that might impact a security baseline?

Reply:
Howe's this?

> The Pod Security Standards are maintained by the K8s
community, which means you automatically get updated security policies whenever new
security-impacting Kubernetes features are introduced.